### DISCUSSION TITLE
Is there any way to hide the right segments when the command is too long?

### DISCUSSION THREAD

**Discussion by rtrindvg** [JanDeDobbeleer/oh-my-posh#1925] [Troubleshoot] [answered]
If you start typing, when you reach the right segments, they are replaced character by character, allowing you to see what is being typed.
![image](https://user-images.githubusercontent.com/82324398/158237118-8592d12b-a5b9-4cae-9989-1a51b55ed654.png)

As soon as the line breaks, if you return to the first line and type anything, the right segments are back, and it's impossible to see which characters are behind them. 
![image](https://user-images.githubusercontent.com/82324398/158237593-a8266bd6-438f-4d06-8240-0a3ad8afc8b5.png)

This also disrupts the copy/paste behavior.

Is this something configurable on the theme? Is there any way to make the theme (or Oh My Posh) ignore this behavior? I prefer to have the right segments gone sometimes than dealing with the current situation.

**Comment by JanDeDobbeleer** [comment, marked as answer]
No, that's not possible unless the shell has a native rprompt functionality (like zsh, fish and cmd).

**Reply by rtrindvg** [reply]
No problem. Since rprompt is what was needed, I just changed my default shell to zsh and the theme is now working perfectly (much better than the closest themes from oh my zsh). 

Thanks for the awesome work on this tool!
